### PR TITLE
[EE-1663] migrate from jenkins to gha

### PR DESCRIPTION
**Pull request by thomaschaplin**

### CODE REVIEW

**Review by jdeastwood — DISMISSED** [review]

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: NPM Publish
+
+on:
+  workflow_dispatch:
```
Comment (akedalat) [inline .github/workflows/npm-publish.yml:4]: Couldn't figure out from the jenkins file but I think this was a post-merge event? if we want to have it manual do you reckon we should add a line to readme?

Comment (thomaschaplin) [reply .github/workflows/npm-publish.yml:4]: Ah yes, sorry this is a manual process now - I'll update the readme with why and how it is done

Comment (akedalat) [reply .github/workflows/npm-publish.yml:4]: Thanks

**Review by simplyzee — DISMISSED** [review]

**Review by akedalat — APPROVED** [review]
LGTM

**Review by jdeastwood — APPROVED** [review]

**Pull request merged**